WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -22,24 +22,52 @@
 </div>
 
 <div class="form-group">
-  <%= label_tag(:years) %>
-  <%= select_tag(:years, options_for_select(@years, selected: params[:years], disabled: ""), class: "form-control") %>
-</div>
-
-<div class="form-group">
   <%= label_tag("Name, City or Venue") %>
   <%= text_field_tag "search", params[:search], class: "form-control" %>
 </div>
 
 <div class="form-group">
-  <label>&nbsp;</label>
-  <div class="btn-group">
-    <%= button_tag(type: 'submit', name: "display", value: "List", class: "btn btn-info") do %>
-      <i class="glyphicon glyphicon-list"></i> List
-    <% end %>
-
-    <%= button_tag(type: 'submit', name: "display", value: "Map", class: "btn btn-primary") do %>
-      <i class="glyphicon glyphicon-map-marker"></i> Map
-    <% end %>
+  <%= label_tag(:state) %>
+  <div id="state" class="btn-group" data-toggle="buttons">
+    <label id="present" class="btn btn-primary">
+      <%= radio_button_tag "state", "present" %>
+      <span class="caption">Present</span>
+    </label>
+    <ul class="dropdown-menu years">
```

Comment:
I checked this on my phone and it's fine. Your suggestion is nice, but I'm not sure if we can achieve it easily. We can always increase the list font size for smaller devices.